drivers/macvlan/macvlan_setup.go

```diff
@@ -74,6 +74,15 @@ func parentExists(ifaceStr string) bool {
 	return true
 }
 
+// vlanLinkExists checks if specified vlan link exists in the default namespace
```

Comment:
same function exist in ipvlan setup code.   can we move into single common place.  its redundant code.

Reply:
I don't think we share any code between Macvlan and ipvlan drivers today, we can cleanup this code in future, aiming to keep the footprint of the change smaller in this PR to only solve the current problem